tests/integration/test_eoc_journal.py

```diff
@@ -224,7 +226,7 @@ def get_element_for_course_engagement(self):
 
     def select_option_in_boolean_field(self, field_name, value=True):
         control = self.get_element_for_field(field_name)
-        select = Select(control.find_element_by_id("xb-field-edit-{}".format(field_name)))
+        select = Select(control)
```

Comment:
Why is this change needed?

Reply:
Somehow ``control`` itself is the ``xb-field-edit-{}`` control, which previously was the control wrapper (``li``) element. This might be due to changes in ``xblock-sdk`` version. But I didn't quite understand why two versions of python getting two different behaviour for ``get_element_for_field`` method.

Reply:
It's likely because of a change in some library. Might be worth investigating.